packages/core/src/Utils/pwa/register-service-worker.js

```diff
@@ -1,4 +1,4 @@
-const EVERY_HOUR = 3600000; // 1000 * 60 * 60
+const EVERY_HOUR = 36000000; // 1000 * 60 * 60
```

Comment:
this is wrong as per the comment value should be what it used to be, it's amounting to an hour now